server/channels/app/platform/session.go

```
@@ -108,7 +116,9 @@ func (ps *PlatformService) ClearUserSessionCache(userID string) {
 }
 
 func (ps *PlatformService) ClearAllUsersSessionCache() {
-	ps.ClearAllUsersSessionCacheLocal()
+	if err := ps.ClearAllUsersSessionCacheLocal(); err != nil {
+		ps.Logger().Error("Failed to purge session cache", mlog.Err(err))
+	}
```

I'm a bit concerned that we don't handle these cases in the same way.

@agnivade Do you have thought about logging cache errors vs returning them>

---

We should return the error. Failing to purge the cache might lead to unexpected behavior.

---

Thanks for the feedback, @hanzei @agnivade 

Just to make sure I'm on the right track: Given that failing to purge the session cache could lead to unexpected behavior,I feel  that I should return the error in such cases to ensure proper handling.

For other less critical errors (like logging failures), I would log the error and continue execution without returning it.

Does this approach sound correct to you? Please let me know if there's anything I should adjust.

Thanks again for your guidance!